mapper/pojo-base/src/main/java/org/hibernate/search/mapper/pojo/massindexing/spi/PojoMassIndexer.java

```diff
@@ -84,7 +84,8 @@ public interface PojoMassIndexer {
 	 * Set this to false only if you know there are no
 	 * entities in the indexes: otherwise search results may be duplicated.
 	 * <p>
-	 * Defaults to {@code true}.
+	 * Default value depends on {@link #dropAndCreateSchemaOnStart(boolean)}. Defaults to {@code false} if the mass indexer
+	 * is configured to drop and create the schema on the start, to {@code true} otherwise.
```

Comment:
```suggestion
	 * Default value depends on {@link #dropAndCreateSchemaOnStart(boolean)}. Defaults to {@code false} if the mass indexer
	 * is configured to drop and create the schema on start, to {@code true} otherwise.
```